### PR TITLE
ActiveRecord 7

### PR DESCRIPTION
**Pull request by keithdoggett**
Support for ActiveRecord 7

TODOs:

- [x]  Create stable branch of version 7
- [x]  ~~Create PostGIS extension migration for Rails apps~~
- [x] Test against ActiveRecord test suite (#358)

### CODE REVIEW

**Comment by seuros** [comment]
The migration is described in the readme. I don't think we should init an engine just for a basic migration.

**Comment by keithdoggett** [comment]
Agreed

**Comment by seuros** [comment]
I will fix it. I will try to reproduce itl

**Comment by keithdoggett** [comment]
Thanks for reporting that! Do we still need a Railtie even if we don't load our own tasks in order to have the Postgres Adapter's tasks?

**Comment by seuros** [comment]
@keithdoggett No need. 
I had to register the tasks only. #355  has the fix.

File: test/attributes_test.rb [hunk 1]
```diff
@@ -71,7 +71,7 @@ def test_spatial_attributes
 
     assert_equal 3857, data.polygon.srid
     assert_equal 3857, data.path.srid
-    assert_equal data.path, data.polygon.exterior_ring
+    assert_equal data.path.to_s, data.polygon.exterior_ring.to_s
```
Comment (keithdoggett) [inline test/attributes_test.rb:74]: noting that this still needs to be looked at after merge

**Comment by keithdoggett** [comment]
Version 8 has been released

**Pull request merged**